crates/cli-support/src/js/mod.rs

```diff
@@ -1748,31 +1744,31 @@ impl<'a, 'b> SubContext<'a, 'b> {
     }
 }
 
-fn indent(s: &str) -> String {
-    let mut ret = String::new();
-    for line in s.lines() {
-        ret.push_str("    ");
-        ret.push_str(line);
-        ret.push_str("\n");
-    }
-    return ret
-}
-
-fn deindent(s: &str) -> String {
-    let amt_to_strip = s.lines()
-        .filter(|l| !l.trim().is_empty())
-        .map(|s| s.len() - s.trim_left().len())
-        .min()
-        .unwrap_or(0);
-    let mut ret = String::new();
-    for line in s.lines() {
-        if !line.trim().is_empty() {
-            ret.push_str(&line[amt_to_strip..]);
-        }
-        ret.push_str("\n");
-    }
-    ret
-}
+// fn indent(s: &str) -> String {
+//     let mut ret = String::new();
+//     for line in s.lines() {
+//         ret.push_str("    ");
+//         ret.push_str(line);
+//         ret.push_str("\n");
+//     }
+//     return ret
+// }
+
+// fn deindent(s: &str) -> String {
+//     let amt_to_strip = s.lines()
+//         .filter(|l| !l.trim().is_empty())
+//         .map(|s| s.len() - s.trim_left().len())
+//         .min()
+//         .unwrap_or(0);
+//     let mut ret = String::new();
+//     for line in s.lines() {
+//         if !line.trim().is_empty() {
+//             ret.push_str(&line[amt_to_strip..]);
+//         }
+//         ret.push_str("\n");
+//     }
+//     ret
+// }
```

Comment:
Let's remove this code if it is not used anymore.

Reply:
oops, sorry about that, this has been removed